Source/backend/models/Product.js

```diff
@@ -0,0 +1,16 @@
+import mongoose from 'mongoose';
+
+const productSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  price: {
+    type: Number,
+    required: true,
+  },
+  description: String,
+  image: String,
+});
+
```

Comment:
Add comments to explain the purpose of each field. This makes the code easier to read and maintain